view/frontend/templates/tag_manager.phtml

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Copyright 2016-2018 Henrik Hedelund
+ * Copyright 2020      Falco Nogatz
+ *
+ * This file is part of Chessio_Matomo.
+ *
+ * Chessio_Matomo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Chessio_Matomo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Chessio_Matomo.  If not, see <http://www.gnu.org/licenses/>.
+ */
+?>
+<?php /** @var \Chessio\Matomo\Block\Matomo $block */
+if (!$block->isContainerEnabled()) {
+    return;
+}
+?>
+<script type="text/x-magento-init">
```

Comment:
In the original PR (that targeted the [fork](https://github.com/chessio/magento2-matomo/pull/1)), @amenk wrote:

> I am wondering if we could extract this, together with the workaround script, to a seperate template to avoid duplication

Reply:
what do you think? should we create an additional block (actually 2 additional blocks) or leave it like that?